File: components_page/components/list_group/links.py
import dash_bootstrap_components as dbc
import dash_html_components as html
from dash.dependencies import Input, Output

list_group = html.Div(
    [
        dbc.ListGroup(
            [
                dbc.ListGroupItem(
                    "Internal link", href="/l/components/list_group"
                ),
                dbc.ListGroupItem("External link", href="https://google.com"),
                dbc.ListGroupItem(
                    "Disabled link", href="https://google.com", disabled=True
                ),
                dbc.ListGroupItem(
                    "Button", id="button-item", n_clicks=0, action=True
                ),
            ]
        ),
        html.P(id="counter"),
    ]
)


@app.callback(
    Output("counter", "children"), [Input("button-item", "n_clicks")]
)
def count_clicks(n):
    return f"Button clicked {n} times"

File: components_page/components/list_group/links.R
library(dashBootstrapComponents)
library(dashHtmlComponents)

list_group <- htmlDiv(
  list(
    dbcListGroup(
      list(
        dbcListGroupItem(
          "Internal link",
          href = "/l/components/list_group"
        ),
        dbcListGroupItem("External link", href = "https://google.com"),
        dbcListGroupItem(
          "Disabled link",
          href = "https://google.com", disabled = TRUE
        ),
        dbcListGroupItem(
          "Button",
          id = "button-item", n_clicks = 0, action = TRUE
        )
      )
    ),
    htmlP(id = "counter")
  )
)


app$callback(
  output("counter", "children"),
  list(input("button-item", "n_clicks")),
  function(n) {
    return(sprintf("Button clicked %d times", n))
  }
)
